docs/enriching-your-data/available-enrichments/custom-javascript-enrichment/writing/index.md

```diff
@@ -111,7 +111,7 @@ For events with no entities attached, `getContexts()` will return `null`. The pa
 
 :::note Derived entities
 
-Currently, derived entities (added by other enrichments) cannot be fetched. `event.getDerived_contexts()` will _always_ return `null`.
+For derived entities (added by other enrichments), you can use `event.getDerived_contexts()` in the same way as above. Note that this is only supported since Enrich 3.8.0 (and Snowplow Micro 1.7.1).
```

Comment:
Maybe note earlier version behaviour here too, rather than just removing it entirely?